app/src/main/java/com/better/alarm/presenter/AlarmDetailsFragment.kt

```diff
@@ -265,6 +266,23 @@ class AlarmDetailsFragment : Fragment() {
         .addTo(disposables)
   }
 
+    private fun onCreateVibrateView() {
+        val mVibrateRow by lazy {
```

Comment:
Using lazy here does not make a difference because it is accessed right away. You can replace it with assignment.